src/assets/translations/pt/main.json

```diff
@@ -1910,5 +1926,22 @@
         "cta": "Crie um"
       }
     }
+  },
+  "nft": {
+    "floorPrice": "Floor Price",
```

Comment:
Forgotten translation?

Reply:
Portuguese translator deliberately left as is, based on his experience.